packages/toolkit/src/lib/react-query-service/model/onSuccessAfterModelMutation.ts

```diff
@@ -93,7 +93,7 @@ export async function onSuccessAfterModelMutation(
             [model.name]: watch,
           }
         : { [model.name]: watch }
-    );
+    ); */
```

Comment:
@orangecoloured We can safely remove these lines since they are old and deprecated code 🦤